bin/propolis-server/src/lib/vm/mod.rs

```diff
@@ -111,6 +111,9 @@ pub(crate) mod state_publisher;
 pub(crate) type DeviceMap =
     BTreeMap<String, Arc<dyn propolis::common::Lifecycle>>;
 
+/// Mapping of NIC identifiers to viona device instance IDs.
+pub(crate) type NetworkInterfaceIds = Vec<(uuid::Uuid, KstatInstanceId)>;
```

Comment:
Since this is semantically a map, should we just go ahead and make a `BTreeMap` out of it?

Reply:
Yeah, I talked to @bnaecker  on this earlier. Since we only really have 8 or some limited number, it made more sense just to use a Vec and iterate vs a map.

Reply:
I'll add a comment.